tests/jerry-test-suite/15/15.08/15.08.02/15.08.02.13/15.08.02.13-031.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2016 Samsung Electronics Co., Ltd.
+// Copyright 2016 University of Szeged
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+assert(isNaN(Math.pow(1, "NaN")));
```

Comment:
Simple NaN is ok. However introducing more never hurts, this is the testing infrastructure.